app/models/wakes/resource.rb

```diff
@@ -22,9 +22,10 @@ class Wakes::Resource < ActiveRecord::Base
   validates :label, :presence => true
 
   def to_s
+    legacy_location_labels = legacy_locations.pluck(:host, :path).map { |x| x.join('') }
```

Comment:
@chrismaddocks I like the instinct towards the performant code. In this particular case, though, because it's only for scripting and console usage, I would rather use DRY code than performant code.